src/screens/OrgPost/OrgPost.tsx

```diff
@@ -110,6 +113,7 @@ function orgPost(): JSX.Element {
           text: postinfo,
           organizationId: currentUrl,
           file: postImage || postVideo,
+          pinned: pinPost,
         },
       });
       /* istanbul ignore next */
```

Comment:
remove this /* Istanbul ignore next */ comment

Reply:
@aashimawadhwa I have removed the comment and the test coverage dropped. To increase the test coverage I was trying to update tests to bring them back to 100%. But every single tests in `src/screens/OrgPost/OrgPost.test.tsx` which calls `CREATE_POST_MUTATION` are throwing error after it is called. The test were written exactly before that point where it starts throwing error, that is why they are currently passing.

The error looks like this: 
`ApolloError: No more mocked responses for the query: mutation CreatePost($text: String!, $title: String!, $organizationId: ID!, $file: String, $pinned: Boolean) {
      createPost(
        data: {text: $text, title: $title, organizationId: $organizationId, pinned: $pinned}
        file: $file
      ) {
        _id
      }
    }
    , variables: {"title":"Dummy Post","text":"This is dummy text","file":"","pinned":false}`

I have tried many solutions, but nothing is actually worked. Please help.

Reply:
@aashimawadhwa Should I comment it out again for now?

Reply:
@pateldivyesh1323 yes, please add the comment here. It just conveys Istanbul to not consider the `if` branch for code coverage. And, I guess it's not possible to test this branch. We have been using this comment in similar conditions in every other part of the codebase. For now, we can continue having this here.